guides/guide-for-adding-windows-node.md

```diff
@@ -1,229 +1,402 @@
 
 # Adding Windows nodes
 
-  
-
 You can use Kubernetes to run a mixture of Linux and Windows nodes, so you can mix Pods that run on Linux on with Pods that run on Windows. This is a guide on how to register Windows nodes to your cluster.
 
-  
 
 ## Before you begin
 
-  
-
 Your Kubernetes server must be at or later than version 1.22. To check the version, enter `kubectl version`.
 
-  
-
 - Obtain a [Windows Server 2019 license](https://www.microsoft.com/en-us/cloud-platform/windows-server-pricing) (or higher) in order to configure the Windows node that hosts Windows containers. If you are using VXLAN/Overlay networking you must have also have [KB4489899](https://support.microsoft.com/help/4489899) installed.
 
 - A Linux-based Kubernetes kubeadm cluster in which you have access to the control plane (see [Creating a single control-plane cluster with kubeadm](https://kubernetes-docsy-staging.netlify.app/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/)).
 
-  
 
 ## Objectives
 
-  
-
 - Register a Windows node to the cluster
-
 - Configure networking so Pods and Services on Linux and Windows can communicate with each other
 
-  
 
 ## Getting Started: Adding a Windows Node to Your Cluster
 
-  
 
 ### Networking Configuration
 
-  
-
 Once you have a Linux-based Kubernetes control-plane node you are ready to choose a networking solution.
 
-  
 
-#### Configuring Flannel with rancher
-
-  
+## Configuring Flannel with rancher
```

Comment:
This should work on any base distro not just Rancher.  I don't see a reference to rancher beyond this.  To make sure folks who aren't using rancher don't turn away we should make this more generic:

```suggestion
## Configuring Flannel on Linux nodes
```
of something similiar?

Reply:
I will update it to be more generic. There is also a flannel hostprocess section and wanted to differentiate them